cluster-install/readme.adoc

```diff
@@ -42,15 +48,19 @@ https://github.com/kubernetes/kops/blob/master/docs/aws.md#setup-iam-user. https
 
 == S3 bucket to store Kubernetes config
 
-Kops needs a "`state store`" to store configuration information of the cluster.  For example, how many nodes in the cluster, the instance type of each node, and the Kubernetes version. The state is stored during the initial cluster creation. Any subsequent changes to the cluster are also persisted to this store. As of now, Amazon S3 is the only supported storage mechanism. Create an S3 bucket and pass that to the kops CLI during cluster creation.
+Kops needs a "`state store`" to store configuration information of the cluster.  For example, how many nodes in the cluster, the instance type of each node, and the Kubernetes version. The state is stored during the initial cluster creation. Any subsequent changes to the cluster are also persisted to this store. As of now, Amazon S3 is the only supported storage mechanism. Create an S3 bucket and pass that to the kops CLI during cluster creation. The bucket name must be unique otherwise you will encounter an error on deployment. For demonstration purposes only, we will use an example bucket name of example-state-store.
+    
+    # create variables for bucket, state store and cluster names
+    export KOPS_STATE_STORE=s3://example-state-store
```

Comment:
Could we do this better using multiple envs? something like:

```
export S3_BUCKET=example-state-store
export KOPS_STATE_STORE=s3://${S3_BUCKET}
```

My thoughts here are so that the bucket versioning you can use also use and env, just for easy of copy paste-age. like:

`aws s3api put-bucket-versioning --bucket $S3_BUCKET --versioning-configuration Status=Enabled`

what do you think?

Reply:
That seems a fair ask. @cloudymind one last change potentially?

Reply:
definitely a good change. i'll modify that